e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js

```diff
@@ -52,8 +52,9 @@ Scenario('Judiciary enters hearing details and submits', async (I, caseViewPage,
   I.seeAnswerInTab(5, 'Hearing 1', '', hearingDetails[0].type.welsh);
   I.seeAnswerInTab(5, 'Hearing 1', '', hearingDetails[0].type.somethingElse);
   I.seeAnswerInTab(6, 'Hearing 1', 'Give details', hearingDetails[0].giveDetails);
-  I.seeAnswerInTab(7, 'Hearing 1', 'Judge or magistrate\'s title', hearingDetails[0].judgeTitle);
-  I.seeAnswerInTab(8, 'Hearing 1', 'Judge or magistrate\'s last name', hearingDetails[0].lastName);
+  I.seeAnswerInTab(1, 'Judge and legal advisor', 'Judge or magistrate\'s title', hearingDetails[0].judgeAndLegalAdvisor.judgeTitle);
```

Comment:
Not your fault but the fact you had to make same change in 3 different files is smell of a bad design / implementation.

Just a note.

Reply:
agree - we do the same test multiple times for different user in different state etc, and we simply copy paste this test in different files. e2e tests are full of duplicate code 😸

Reply:
@akomisarek, you might look into these duplicates in e2e tests. I question whether it makes sense to test same thing many times for each role eligible for action. Wouldn't it be enough to test it once to make sure UI it correct and then separately make sure that configuration has right permissions for roles or that roles inheritance is correct?

Not trivial problem but wanted to highlight it as I am not convinced with current approach.